CMakeLists.txt

```diff
@@ -22,15 +22,20 @@ cmake_minimum_required(VERSION 3.10...3.17.2)
 
 project(OpenSSL VERSION "1.0.0" LANGUAGES CXX DESCRIPTION "CMake to compile OpenSSL" HOMEPAGE_URL "https://github.com/flagarde/openssl-cmake")
 
-set(OPENSSL_TAG "1_1_1i" CACHE STRING "Version the OpenSSL to download")
+if(OPENSSL_VER)
+  string(REPLACE "." "_" OPENSSL_TAG ${OPENSSL_VER})
+else()
+   set(OPENSSL_TAG "1_1_1j" CACHE STRING "Version the OpenSSL to download")
+endif()
+
 set(OPENSSL_URL "https://github.com/openssl/openssl/archive/OpenSSL_${OPENSSL_TAG}.zip" CACHE STRING "URL of the OpenSSL depot")
 
 if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
   message(FATAL_ERROR "In-source builds are not allowed")
 endif()
 
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-  set(CMAKE_INSTALL_PREFIX "${PROJECT_SOURCE_DIR}/bin" CACHE PATH "Default Install Prefix" FORCE)
+  set(CMAKE_INSTALL_PREFIX "${PROJECT_SOURCE_DIR}/openssl-${OPENSSL_TAG}" CACHE PATH "Default Install Prefix" FORCE)
```

Comment:
I would prefer 
```cmake
set(CMAKE_INSTALL_PREFIX "${PROJECT_SOURCE_DIR}/bin/openssl-${OPENSSL_TAG}" CACHE PATH "Default Install Prefix" FORCE)
```
By default I like to put all in bin to avoid mixing the sources with the bin folders.

Reply:
Ok reason for change was OpenSSL also creates a bin folder so they were nested inside each other

Reply:
oh ok Nice :)